client/components/domains/domain-registration-suggestion/utility.js

```diff
@@ -0,0 +1,63 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { includes } from 'lodash';
+
+export const newTLDs = [
```

Comment:
Maybe let's not `export` these two lists - so that no one is tempted to use them directly.